android/src/main/java/com/expensify/livemarkdown/MarkdownRange.java

```diff
@@ -0,0 +1,18 @@
+package com.expensify.livemarkdown;
+
+public class MarkdownRange {
+  public String type;
+  public int start;
+  public int end;
+  public int length;
+  public int depth;
+
+  public MarkdownRange(String type, int start, int end, int depth) {
+    this.type = type;
+    this.start = start;
+    this.end = end;
+    this.length = end - start + 1;
+    this.depth = depth;
```

Comment:
```suggestion
    this.type = type;
    this.start = start;
    this.length = length;
    this.end = start + length;
    this.depth = depth;
```

Reply:
This is not resolved

Reply:
Forgot to push some changes, now it should be resolved